### ISSUE TITLE
apply the encumbered effect once.

### ISSUE THREAD

**Issue by drdwing** [VanirDev/VariantEncumbrance#46]
![Cap 2021-10-17 16-56-16-352](https://user-images.githubusercontent.com/66671688/137621177-39105b23-e328-42fb-a8ee-998cb8f1ec6c.jpg)

When somebody get encumbered, status announce is repeatedly displayed.

I think this module keep creating that effect.

Fvtt v0.8.9(chrome)
window 10
dnd 5e v1.5.2
mod v0.5.19.

**Comment by p4535992** [comment]
The chat is written by the module "https://github.com/DFreds/dfreds-convenient-effects". I couldn't replicate the error there must be some mechanism on your side. Make sure there isn't some automated mechanism that adds / deletes / removes items. For example a bow that removes ammo?

If you use other modules listed it could be an incompatibility between modules that I have escaped.

**Comment by drdwing** [comment]
Ok. Those messages were displayed when players got items using "Loot sheet NPC 5e".
I tried to see the error message by myself but it didn't. I'll try again in a few hours with players and find the cause of error.

**Comment by drdwing** [comment]
Oh, I think I found it.
The message is repeated when a user moves an item while a character is encumbered(using Inventory + mod).
I think a sheet is refreshed by moving an item, so convenient effects display effect messages whenever a user moved an item.
And...I solved this problem by changing the permission of applied effect chat message. 😢
I don't think there is any problem with this module. Sorry!!
